The linker may not understand this flag:
ld: unknown option: -no_fixup_chains
collect2: error: ld returned 1 exit status

--- configure	2024-05-25 21:26:49.000000000 +0800
+++ configure	2024-05-30 14:02:19.000000000 +0800
@@ -8761,10 +8761,11 @@
       _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
     darwin*)
       case $MACOSX_DEPLOYMENT_TARGET,$host in
+        # This is correct, later PowerPC uses -undefined dynamic_lookup
         10.[012],*|,*powerpc*-darwin[5-8]*)
           _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
         *)
-          _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup $wl-no_fixup_chains' ;;
+          _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup' ;;
       esac
     ;;
   esac
